src/test/java/seedu/academyassist/logic/commands/DetailCommandTest.java

```diff
@@ -0,0 +1,16 @@
+package seedu.academyassist.logic.commands;
+
+import static seedu.academyassist.testutil.TypicalPersons.getTypicalAcademyAssist;
+
+import seedu.academyassist.model.Model;
+import seedu.academyassist.model.ModelManager;
+import seedu.academyassist.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
```

Comment:
Good idea to include integration tests for DetailCommand. Can consider extracting out integration and unit tests into separate files, since that's what was done for AddCommand